### PR TITLE
[esc] update pulumi-cloud docs for projects

### PR DESCRIPTION
**Pull request by dschaller**
### Proposed changes

updates pulumi-cloud docs for ESC projects

### CODE REVIEW

File: content/docs/pulumi-cloud/cloud-rest-api/_index.md [hunk 1]
```diff
@@ -3251,16 +3251,12 @@ Kubernetes Pods should not be used directly. Instead, you may want to use a Depl
 
 ## Environments
 
-{{< notes >}}
-Pulumi ESC (Environments, Secrets, and Configuration) and its associated REST API endpoints are currently in public preview.
-{{< /notes >}}
```
Comment (komalali) [inline content/docs/pulumi-cloud/cloud-rest-api/_index.md:3256]: I think for this we're actually going to need to do something similar to https://www.pulumi.com/docs/pulumi-cloud/deployments/api/#migrating-from-the-preview-api and we'll also need to add the note about ESC GA and how the preview routes are being deprecated (but that should go out next Wednesday).

But the migration guide should be necessary coz I think we changed some of the payloads from the preview routes.

File: content/docs/pulumi-cloud/cloud-rest-api/_index.md [hunk 2]
```diff
@@ -3303,15 +3313,21 @@ Status: 200 OK
 ### Create Environment
 
 ```
-POST /api/preview/environments/{organization}/{environment}
+POST /esc/environments/{organization}/
```
Comment (seanyeh) [inline content/docs/pulumi-cloud/cloud-rest-api/_index.md:3316]: np no trailing slash

Comment (dschaller) [reply content/docs/pulumi-cloud/cloud-rest-api/_index.md:3316]: 🤦 you are right!

**Comment by dschaller** [comment]
> OH! almost missed this. I think we still keep /api portion?

🤦 you are right!

**Pull request merged**